google/cloud/forseti/services/model/importer/importer.py

```diff
@@ -536,25 +544,37 @@ def _convert_appengine_resource(self, gae_resource):
         self.session.add(resource)
         self._add_to_cache(gae_resource, resource)
 
-    def _convert_dataset(self, dataset):
+    def _convert_dataset(self, dataset, policy):
```

Comment:
I don't think it is a good idea to deal with dataset and dataset_policy at the same time. It is a one off and temporary solution. I would suggest using the same flow as iam_policy.